packages/browser/src/Component/Widget/Server/SPARQL/Readme.md

```diff
@@ -108,6 +78,32 @@ The JSON view returns a collapsible representation of the query reponse.
 
 ![JSON view demonstraction](/docs/static/Widget/Extensions/SPARQL/Pictures/sparql-widget-json-demo.gif)
 
+### Widget Configuration
+
+The minimal configuration required to make the SPARQL module work is the following :
+
+```json
+{
+  "sparqlModule": {
+    "height": 500,
+    "width": 500,
+    "fontSize": 4,
+    "url": "http://localhost:9999/strabon/",
+    "url_parameters": "Query?handle=download&format=SPARQL/JSON&view=HTML&query="
+  }
+}
+```
+- `height` the height (in pixels) of a d3 canvas for visualizing graphs
+- `width` the width (in pixels) of a d3 canvas for visualizing graphs
+- `fontSize` fontsize to be used in the d3 canvas for labeling graph nodes
+- `sparqlModule.url` represents the base URL for the server. 
```

Comment:
remove sparqlModule. to be symetric with above

Reply:
updated readme to match